### PR TITLE
WIP:[Q-0011]

### PR DESCRIPTION
**Pull request by daart**
Work is still in Progress do not  #Merge!!

-kill game when game is over in 4 seconds, and when a last player left the game

### CODE REVIEW

**Comment by daart** [comment]
#16 #17

File: client/src/components/Game/Game.tsx [hunk 1]
```diff
@@ -94,10 +94,21 @@ const Game = () => {
     const onPlayerLeft = message => {
       console.log(message);
     };
+
+    const onGameOver = message => {
+      console.log('It is over biatch!: ', message);
+      setAppState({
```
Comment (bigjey) [inline client/src/components/Game/Game.tsx]: Should we also reset gameInfo?

File: server/Game.ts [hunk 2]
```diff
@@ -182,6 +192,14 @@ export class Game {
       }
     }
 
+    if (player.socketId in io.sockets.sockets) {
+      io.sockets.sockets[player.socketId].leave(this.id);
+    }
+
+    if (this.id && !Object.keys(this.players).length) {
```
Comment (bigjey) [inline server/Game.ts]: why do we check for `this.id`?

File: server/Game.ts [hunk 3]
```diff
@@ -328,6 +369,8 @@ export class Game {
           };
         })
         .sort((pr1, pr2) => pr2.score - pr1.score);
+
+      this.startKickAllPlayersCountdown();
```
Comment (bigjey) [inline server/Game.ts]: i would rather start it in the place that sets game stage to GAME_OVER than in updateGameInfo method, that one can be called multiple times

**Comment by daart** [comment]
closed due to inactuality

**Pull request closed without merging**